Catalog.html

```diff
@@ -0,0 +1,362 @@
+                  <img
+                    class="rating-card-image"
+                    src="#"
+                    alt="Фото  рейтинги Villas at Poco Diablo"
+                  />
+                  <p>Рейтинг: 9,2</p>
+                </div>
+              </div>
+            </li>
+            <li class="catalog-item">
+              <div class="product-card">
+                <div class="catalog-product">
+                  <a class="product-card-link" href="product.html">
+                    <img
+                      class="product-card-image"
+                      src="#"
+                      alt="GreenTree Inn"
+                    />
+                    <h3 class="product-card-title">Villas at GreenTree Inn</h3>
+                  </a>
+                  <p class="product-card-price">Гостиница от 3000 ₽</p>
+                  <a href="#" class="product-card-button detailed">Подробнее</a>
+                  <a href="#" class="product-card-button favourites"
+                    >Избранное</a
+                  >
+                </div>
+                <div class="rating-product">
+                  <img
+                    class="rating-card-image"
+                    src="#"
+                    alt="Фото  рейтинги GreenTree Inn"
+                  />
+                  <p>Рейтинг: 9,2</p>
+                </div>
+              </div>
+            </li>
+          </ul>
+          <a href="#" class="more product">Загрузить еще</a>
+        </article>
+        <footer>
+          <ul class="pagination">
+            <li class="pagination-item">
+              <a class="pagination-link pagination-prev pagination-disabled">
+                <span class="visually-hidden">Предыдущая страница</span>
+              </a>
+            </li>
+            <li class="pagination-item">
+              <a class="pagination-link pagination-current">1</a>
+            </li>
+            <li class="pagination-item">
+              <a class="pagination-link" href="#">2</a>
+            </li>
+            <li class="pagination-item">
+              <a class="pagination-link" href="#">3</a>
+            </li>
+            <li class="pagination-item">
+              <a class="pagination-link" href="#">4</a>
+            </li>
+            <li class="pagination-item">
+              <a class="pagination-link" href="#">5</a>
+            </li>
+            <li class="pagination-item">
+              <a class="pagination-link pagination-next" href="#">
+                <span class="visually-hidden">Следующая страница</span>
+              </a>
+            </li>
+          </ul>
+          <!--Вот тут нужна помощь, не совсем поняла как реализовать-->
+          <label class="show">Гостиниц на странице</label>
+          <select class="show-list">
+            <option class="show-item">4</option>
+            <option class="show-item">8</option>
+            <option class="show-item">16</option>
+          </select>
```

Comment:
нужно связать лейбл с выбором. Либо вложив
```suggestion
          <label class="show">Гостиниц на странице
            <select class="show-list">
              <option class="show-item" selected>4</option>
              <option class="show-item">8</option>
              <option class="show-item">16</option>
            </select>
          </label>
```
Либо с помощью id
```suggestion
          <label class="show" for="show-list">Гостиниц на странице</label>
          <select class="show-list" id="show-list">
            <option class="show-item">4</option>
            <option class="show-item">8</option>
            <option class="show-item">16</option>
          </select>
```

Reply:
Исправила